Swift Vocabulary/Swift Vocabulary/View Controllers/WordsTableViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+
+    var words: [VocabularyWord] = [VocabularyWord(word: "Variable", definition: "Variable Definition"),
+    VocabularyWord(word: "Constant", definition: "Constant Definition"),
+    VocabularyWord(word: "Function", definition: "Function Definition")
+    ]
+  
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return words.count
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "WordCell", for: indexPath)
+
+        cell.textLabel?.text = words[indexPath.row].word
+
+        return cell
+    }
+    
+
+    /*
+    // Override to support conditional editing of the table view.
+    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        // Return false if you do not want the specified item to be editable.
+        return true
+    }
+    */
+
+    /*
+    // Override to support editing the table view.
+    override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            // Delete the row from the data source
+            tableView.deleteRows(at: [indexPath], with: .fade)
+        } else if editingStyle == .insert {
+            // Create a new instance of the appropriate class, insert it into the array, and add a new row to the table view
+        }    
+    }
+    */
+
+    /*
+    // Override to support rearranging the table view.
+    override func tableView(_ tableView: UITableView, moveRowAt fromIndexPath: IndexPath, to: IndexPath) {
+
+    }
+    */
+
+    /*
+    // Override to support conditional rearranging of the table view.
+    override func tableView(_ tableView: UITableView, canMoveRowAt indexPath: IndexPath) -> Bool {
+        // Return false if you do not want the item to be re-orderable.
+        return true
+    }
+    */
+
+    
+    // MARK: - Navigation
+
+    // In a storyboard-based application, you will often want to do a little preparation before navigation
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "ShowDefinitionSegue" {
+            guard let indexPath = tableView.indexPathForSelectedRow,
+                let defintionVC = segue.destination as? DefinitionViewController else {return}
+            
+            defintionVC.vocabWord = words[indexPath.row]
+        }
+        
+    }
```

Comment:
Nice job unwrapping and casting this! And like I said, the VC abbreviation is just fine here.